packages/storybook/package.json

```diff
@@ -25,39 +25,27 @@
     "@ni/nimble-tokens": "*",
     "@ni/spright-components": "*",
     "@ni/spright-react": "*",
-    "@rollup/plugin-node-resolve": "^16.0.0",
-    "@storybook/addon-a11y": "^8.4.3",
-    "@storybook/addon-actions": "^8.4.3",
-    "@storybook/addon-docs": "^8.4.3",
-    "@storybook/addon-essentials": "^8.4.3",
-    "@storybook/addon-interactions": "^8.4.3",
-    "@storybook/addon-links": "^8.4.3",
-    "@storybook/addon-webpack5-compiler-swc": "^3.0.0",
-    "@storybook/cli": "^8.4.3",
-    "@storybook/components": "^8.6.12",
+    "@storybook/addon-a11y": "9.0.5",
+    "@storybook/addon-docs": "9.0.5",
+    "@storybook/addon-links": "9.0.5",
+    "@storybook/cli": "9.0.5",
     "@storybook/csf": "^0.1.11",
```

Comment:
don't think this is needed anymore based on the comment on the repo: https://github.com/ComponentDriven/csf

> NOTE This repo is inactive. The code from this repo lives in the [Storybook monorepo](https://github.com/storybookjs/storybook/tree/next/code/core/src/csf) and is no longer published as a standalone package. If you depend on the @storybook/csf package directly, please file an issue on this repo explaining your use case. We'll be figuring out a longer-term solution over the coming months.